docs/guides/using-watt-with-node-config.md

```diff
@@ -0,0 +1,133 @@
+# Using Watt With Node Config
+
+[Node-config](https://www.npmjs.com/package/config) is a powerful configuration management package that helps organize settings across different deployment environments in your application. It creates a unified configuration system that works seamlessly with both [Watt](https://platformatic.dev/docs/watt/overview) and other `npm` modules.
+
+## Benefits of node-config
+
+[Node-config](https://www.npmjs.com/package/config) provides a flexible hierarchy for your application settings. You can define default values and then customize them for specific environments like development, QA, staging, and production. The configuration values can come from multiple sources, including configuration files, environment variables, command line parameters, and external services.
+
```

Comment:
I would not promote this in any form. There are _no benefits_ in using this module. In fact, it's the source of confusion for a lot of people.

https://nodejs.org/en/learn/getting-started/nodejs-the-difference-between-development-and-production#why-is-node_env-considered-an-antipattern
https://www.youtube.com/watch?v=HMM7GJC5E2o&t=57s